app/pages/nonsipp/membersurrenderedbenefits/SurrenderedBenefitsCompletedPage.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.membersurrenderedbenefits
+
+import config.Refined._
+import models.SchemeId.Srn
+import pages.{IndexedQuestionPage, QuestionPage}
+import play.api.libs.json.JsPath
+import utils.RefinedUtils._
+import viewmodels.models.SectionCompleted
+
+case class SurrenderedBenefitsCompletedPage(srn: Srn, index: Max300) extends QuestionPage[SectionCompleted.type] {
+
+  override def path: JsPath = JsPath \ toString \ index.arrayIndex.toString
```

Comment:
Regarding both paths, would it be better if we keep them under it's relevant parent.

I mean like this `Paths.memberPensionSurrender \ toString`